django_socio_grpc/utils/registry_singleton.py

```diff
@@ -247,34 +199,19 @@ def get_proto_type(
         if isinstance(field_type, str):
             return field_type
 
-        proto_type = self.type_mapping.get(field_type.__class__.__name__, "string")
-
-        # if  field_name == "slug_many_many":
```

Comment:
can you let keep me this ? it's very useful when debugging event if it's really ugly